R/reporter-junit.R

```diff
@@ -85,22 +85,27 @@ JunitReporter <- R6::R6Class("JunitReporter",
     },
 
     end_context = function(context) {
+      # Make sure that any options posed by particular tests environments
+      # will not interfere with how values are saved to the file
+      withr::local_options(list(OutDec = "."))
       xml2::xml_attr(self$suite, "tests") <- as.character(self$tests)
       xml2::xml_attr(self$suite, "skipped") <- as.character(self$skipped)
       xml2::xml_attr(self$suite, "failures") <- as.character(self$failures)
       xml2::xml_attr(self$suite, "errors") <- as.character(self$errors)
       #jenkins junit plugin requires time has at most 3 digits
       xml2::xml_attr(self$suite, "time") <- as.character(round(self$suite_time, 3))
-
+      
```

Comment:
Could you please get rid of these spurious whitespace changes?

Reply:
Done! Thanks